Makefile

```diff
@@ -356,6 +356,26 @@ supplier2_unstake: ## Unstake supplier2
 supplier3_unstake: ## Unstake supplier3
 	SUPPLIER=supplier3 make supplier_unstake
 
+###############
+### Session ###
+###############
+
+.PHONY: get_session
+get_session: ## Retrieve the session given the following env vars: (APP_ADDR, SVC, HEIGHT)
+	pocketd --home=$(POCKETD_HOME) q session get-session $(APP) $(SVC) $(HEIGHT) --node $(POCKET_NODE)
+
+.PHONY: get_session_app1_anvil
+get_session_app1_anvil: ## Retrieve the session for (app1, anvil, latest_height)
+	APP=pokt1mrqt5f7qh8uxs27cjm9t7v9e74a9vvdnq5jva4 SVC=anvil HEIGHT=0 make get_session
```

Comment:
Wdyt about having a make target which looks up the addresses by name with something like `poktrolld keys show -a <name>`?

I don't have a strong opinion but I think it MAY save us some confusion down the road. Could just be a `TODO_CONSIDERATION` comment or something for now.